internal/pkg/composable/providers/host/host_test.go

```diff
@@ -167,3 +168,20 @@ func returnHostMapping(log *logger.Logger) infoFetcher {
 		return host, nil
 	}
 }
+
+func TestGetHostInfoReturnsSomeKeys(t *testing.T) {
+	// this is a simple test to ensure the host provider returns the new keys
+	// needed to add support to Debian 12.
+	osInfo, err := getHostInfo(logp.L())()
```

Comment:
There is a test logger on https://github.com/elastic/elastic-agent/blob/d8125be16f8bf369738113861f958c59441bc696/pkg/core/logger/loggertest/logger.go#L20. Better to use it. Or at least the logger from `pkg/core/logger/`